HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,103 @@
+
+mpg %>% 
+  ggplot(mapping = aes(x = cty, y = hwy, color = "blue")) +
+  geom_point()
+```
+
+### Graph Fail 2
+Why aren't the points blue? It is making me blue that the points in the graph aren't blue :`(
+
+`Answer:` `color = "blue"` should be outside of the aesthetics. 
+```{r}
+ggplot(data = mpg) + 
+  geom_point(mapping = aes(x = displ, y = hwy), color = "blue")
+
+```
+
+
+
+### Graph Fail 3
+Two mistakes in this graph. First, I wanted to make the the points slightly bolder, but changing the alpha to 2 does nothing. What does alpha do and what does setting it to 2 do? What could be done instead if I want the points slightly bigger? 
+
+`Answer:`We should set the size to make the points seem bolder, alpha determines the transperancy. 
+
+Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+
+`Answer:`The coordinates on each axis actually ranges from 0 to 1, so we should use relative values instead of absolute values. We can remove the legend title by setting legnd.title to element_blank().
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 2) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = c(5/7, 40/45), legend.title = element_blank())
+```
+
+`Answer:`To remove the plot legend completely, we can set legend.position to "none". 
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), size = 2) + 
+  theme(legend.position = "none")
+```
+
+### Graph Fail 4
+I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
+
+`Answer:``"color = drv"` partitioned the data by drv, and geom_point() added this extra layer of scatter plot. 
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_smooth(se = F) #se = F makes it so it won't show the error in the line of fit
+```
+
+### Graph Fail 5
+I got tired of the points, so I went to boxplots instead. However, I wanted the boxes to be all one color, but setting the color aesthetic just changed the outline? How can I make the box one color, not just the outline? 
+
+`Answer:`Fill needs to be changed as well, and color itself only changes the outline. 
+
+Also, the x-axis labels were overlaping, so I rotated them. But now they overlap the bottom of the graph. How can I fix this so axis labels aren't on the graph?
+
+`Answer:`We can change hjust to a larger value to make sure that the axis labels are not on the graph.
+
+```{r}
+ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer, fill = manufacturer)) + 
+  geom_boxplot() + 
+  theme(axis.text.x = element_text(angle = 45, hjust = 1))
```

Comment:
I used the same method. I wonder if we can also achieve the same effect by using vjust?